runtime/moonbase/src/governance/origins.rs

```diff
@@ -11,8 +11,6 @@
 // MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 // GNU General Public License for more details.
 
-#![cfg_attr(not(feature = "std"), no_std)]
-
```

Comment:
Is this relevant?

Reply:
this should be on the top of lib.rs, and it will emit the warnings.